sysutils/pfSense-upgrade/files/pfSense-upgrade

```diff
@@ -567,11 +567,41 @@ pkg_upgrade() {
 		repository_url=$(/usr/local/sbin/read_xml_tag.sh string system/gitsync/repositoryurl)
 		branch=$(/usr/local/sbin/read_xml_tag.sh string system/gitsync/branch)
 
+		minimal=$(/usr/local/sbin/read_xml_tag.sh boolean system/gitsync/minimal)
+		diff=$(/usr/local/sbin/read_xml_tag.sh boolean system/gitsync/diff)
+		show_files=$(/usr/local/sbin/read_xml_tag.sh boolean system/gitsync/show_files)
+		show_command=$(/usr/local/sbin/read_xml_tag.sh boolean system/gitsync/show_command)
+		dryrun=$(/usr/local/sbin/read_xml_tag.sh boolean system/gitsync/dryrun)
+
+		mute='mute'
+		ignore_result='ignore_result'
+
+		options=''
+		if [ "${minimal}" = "true" ]; then
+			options=${options}' --minimal'
+		fi
+		if [ "${diff}" = "true" ]; then
+			options=${options}' --diff'
+		fi
+		if [ "${show_files}" = "true" ]; then
+			options=${options}' --show-files'
+			mute=''
+			ignore_result=''
+		fi
+		if [ "${show_command}" = "true" ]; then
+			options=${options}' --show-command'
+			mute=''
+			ignore_result=''
```

Comment:
Disabling ignore_result will make pfSense-upgrade to crash if gitsync fails, what is not a good idea IMO